pkg/storaged/fsys-tab.jsx

```diff
@@ -253,12 +257,14 @@
                                                     if (mounted_at == new_dir)
                                                         return;
                                                     return (undo()
-                                                            .then(() => client.mount_at(block, old_dir))
-                                                            .then(() => Promise.reject(error))
+                                                            .then(() => {
+                                                                if (is_filesystem_mounted)
+                                                                    return client.mount_at(block, old_dir);
+                                                            })
                                                             .catch(ignored_error => {
                                                                 console.warn("Error during undo:", ignored_error);
-                                                                return Promise.reject(error);
-                                                            }));
+                                                            })
+                                                            .then(() => Promise.reject(error)));
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18180-20230117-101932-3c5e4716-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)